versioned_plugins.rb

```diff
@@ -413,16 +410,15 @@ def lazy_create_output_folder(output_asciidoc)
   # param plugin_names_by_type: map of lists {:input => [beats, tcp, ...]}
   # return list of triples (type, alias, target) es: ("input", "agent", "beats")
   def load_alias_definitions_for_target_plugins(plugin_names_by_type)
-    alias_url = URI('https://raw.githubusercontent.com/elastic/logstash/master/logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml')
-    alias_yml = Net::HTTP.get(alias_url)
-    yaml = YAML::safe_load(alias_yml) || {}
+    alias_plugin_util = Util::AliasPluginUtil.new
+    yaml = alias_plugin_util.fetch_alias_mappings
 
     aliases = []
 
     yaml.each do |type, alias_defs|
-      alias_defs.each do |alias_name, target|
-        if plugin_names_by_type.fetch(type).include?(target)
-          aliases << [type, alias_name, target]
+      alias_defs.each do |alias_definition|
+        if plugin_names_by_type.fetch(type).include?(alias_definition["from"])
+          aliases << [type, alias_definition["alias"], alias_definition["from"]]
```

Comment:
Similar to the other file's use, I would prefer this re-indexing the retrieved mapping to get pulled up into `Util::AliasPluginUtil`.

Reply:
In `Util::AliasDefinitionsLoader` (was `Util::AliasPluginUtil`), provided only get (YAML to domain level Hash) alias definitions and all other service level (`versioned_plugins.rb`, `plugindocs.rb`) logics manipulates based on their requirements.